<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>751</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="graph_verticalLayout"/>
  </widget>
  <widget class="QLabel" name="current_time_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>현재 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="final_result">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>가공 시간: 0월 0일 00시 00분 00초 ~ 0월 0일 00시 00분 00초
0분 0초간의 공정에서 0분 0초에 구멍이 뚫렸습니다</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_time_label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>경과한 시간:</string>
   </property>
  </widget>
  <widget class="QLabel" name="elapsed_time_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="final_result_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>570</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>최종 결과 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="current_time_label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>현재 상태:</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_state_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>xx 구간 가공 중입니다..</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
